bin/cli.js

```diff
@@ -202,7 +206,27 @@ const injectAccountIdMiddleware = async options => {
   if (options.useEnv && process.env.HUBSPOT_ACCOUNT_ID) {
     options.derivedAccountId = parseInt(process.env.HUBSPOT_ACCOUNT_ID, 10);
   } else {
-    options.derivedAccountId = getAccountId(account);
+    try {
+      options.derivedAccountId = getAccountId(account);
+    } catch (error) {
+      logger.error(error.message || error);
+      if (error.cause === DEFAULT_ACCOUNT_OVERRIDE_ERROR_INVALID_ID) {
+        logger.log(
+          i18n(`${i18nKey}.injectAccountIdMiddleware.invalidAccountId`, {
+            overrideCommand: uiCommandReference('hs accounts create-override'),
```

Comment:
nit, but we prefer the singular usage now `hs account create-override`

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1349/commits/5b758d2e4b814cb8abc89bba8cb54cb3c25df7b1